test/online_test.py

```diff
@@ -52,11 +54,16 @@ def setup_module(module):
 
 
 def teardown_module(module):
-    config_title = test_config.get('Spreadsheet', 'title')
-    sheets = pygsheet_client.open_all(query="name = '{}'".format(config_title))
+    sheets = pygsheet_client.open_all()
```

Comment:
Why are you deleting all the sheets?

Reply:
I do the testing on my personal account and this just deleted all my sheets :-1:

Reply:
I am really sorry! I changed this because of the additionally created test sheets when interrupting the tests, or when the class teardown has an error etc. 

I wanted to change if to be more specific, but then forgot about it. I hope you could restore your sheets!

Reply:
Next time please write an elaborate TODO (danger) comment in such cases !!!